api-spec/protobuf/tdexa/v1/analytics.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package tdexa.v1;
+
+/**
+ * Analytics service spec
+ */
+service Analytics {
```

Comment:
Is it this `Market` related to single provider URL?  then how we can switch between other providers? should be part of request eventually.

Reply:
Would be also useful to aggregate the same markets accross different providers in a single API, maybe we can make an average between them  for the price and sum of whole for the balance

Reply:
> Is it this Market related to single provider URL? then how we can switch between other providers? should be part of request eventually.

I was thinking maybe to have market data in internal db, like one table with id, base, quote hash, provider url.

> Would be also useful to aggregate the same markets accross different providers in a single API, maybe we can make an average between them for the price and sum of whole for the balance 

Can we create issue  for this and implement it after we have base apis done?